### ISSUE TITLE
Can i change the text color of calender?

### ISSUE THREAD

**Issue by zamargab** [aleksanderwozniak/table_calendar#593]
Was wondering if it currently possible to change the text color of the calender dates

**Comment by aleksanderwozniak** [comment]
Of course, use `CalendarStyle` or `CalendarBuilders` for this - see the [docs](https://pub.dev/documentation/table_calendar/latest/table_calendar/table_calendar-library.html) for more.
